### DISCUSSION TITLE
@defer

### DISCUSSION THREAD

**Discussion by Sebastian-G** [help-me-mom/ng-mocks#7587] [Q&A] [answered]
Does the `componentFixture.getDeferBlocks()` work as expected? In my test case, it seems to always return an empty list.

Has anyone already gained some experience with deferable views and testing?

**Comment by satanTime** [comment]
Please follow on https://github.com/help-me-mom/ng-mocks/issues/7742, support of `componentFixture.getDeferBlocks()` will be implemented there.

**Comment by satanTime** [comment, marked as answer]
This was implemented by [#14515](https://github.com/help-me-mom/ng-mocks/pull/14515) and released in [v14.16.0](https://github.com/help-me-mom/ng-mocks/releases/tag/v14.16.0). `ComponentFixture.getDeferBlocks()` is now covered when ng-mocks declarations are used in the test.

Closing as resolved.
